packages/contract-wrappers/package.json

```diff
@@ -39,6 +39,7 @@
     },
     "devDependencies": {
         "@0x/contracts-test-utils": "^3.1.2",
+        "@0x/coordinator-server": "^0.0.6",
```

Comment:
It looks like this is bringing in some old dependencies of contract-wrappers, might result in some strange behaviour.